packages/api-reference/src/hooks/useParser.ts

```diff
@@ -65,7 +65,9 @@ export function useParser({
       return
     }
 
-    parse(value)
+    const processedValue = value.trim()
```

Comment:
I’ve run into issues with some kind of additional whitespace sometimes. That fixed it.